examples/todos/src/containers/VisibleTodoList.js

```diff
@@ -15,17 +15,15 @@ const getVisibleTodos = (todos, filter) => {
   }
 }
 
-const mapStateToProps = (state) => ({
+const mapStateToProps = state => ({
   todos: getVisibleTodos(state.todos, state.visibilityFilter)
 })
 
-const mapDispatchToProps = {
-  onTodoClick: toggleTodo
-}
+const mapDispatchToProps = dispatch => ({
+  toggleTodo: id => dispatch(toggleTodo(id))
```

Comment:
i don't understand the purpose if doing this. Yes, it's working, but that's the benefit of doing this over just sending the parameters when calling the action _directly_?